constraints/src/main/java/gov/nasa/jpl/aerie/constraints/tree/ViolationsOf.java

```diff
@@ -18,10 +19,10 @@ public ViolationsOf(Expression<Windows> expression) {
   }
 
   @Override
-  public List<Violation> evaluate(SimulationResults results, Map<String, ActivityInstance> environment) {
-    final var bounds = new Windows(results.bounds);
+  public List<Violation> evaluate(SimulationResults results, final Window bounds, Map<String, ActivityInstance> environment) {
+    final var boundsW = new Windows(bounds);
     final var satisfiedWindows = this.expression.evaluate(results, environment);
```

Comment:
🔻 Looks like we might have missed one call to `evaluate`.

```suggestion
    final var satisfiedWindows = this.expression.evaluate(results, bounds, environment);
```